presto-spi/src/main/java/com/facebook/presto/spi/LocalProperty.java

```diff
@@ -34,6 +34,11 @@
     <T> Optional<LocalProperty<T>> translate(Function<E, Optional<T>> translator);
 
     /**
+     * Returns true if reordering breaks this LocalProperty
```

Comment:
I'm not sure I understand what this means. If "reordering" of what?

Reply:
Reordering of rows. I didn't write "rows" in the javadoc because the `LocalProperty` class seems to abstract what is it local property of.